scanner/matcher/updater/vuln/updater.go

```diff
@@ -479,7 +479,10 @@ func (u *Updater) fetch(ctx context.Context, prevTimestamp time.Time) (*os.File,
 			return nil, time.Time{}, err
 		}
 		req.Header.Set(ifModifiedSince, prevTimestamp.Format(http.TimeFormat))
-
+		// Request multi-bundle if multi-bundle is enabled.
+		if features.ScannerV4MultiBundle.Enabled() {
+			req.Header.Set("X-Scanner-V4-Accept", "application/vnd.stackrox.scanner-v4.multi-bundle+zip")
```

Comment:
Only found out about this due to another PR, but it looks like the `X-` prefix is not recommended for private purposes. Seems like `X.` is preferred https://datatracker.ietf.org/doc/html/rfc6838#section-3.4

Reply:
apologies that RFC is for the media type. This one deprecates the the `X-` convention: https://datatracker.ietf.org/doc/html/rfc6648

That being said, I'm sure this is fine as-is, but I think the community (ie whoever wrote this RFC) may prefer something like `StackRox-Scanner-V4-Accept` (makes it clear this is for StackRox purposes and can't be confused for some other Scanner V4. https://datatracker.ietf.org/doc/html/rfc6648#section-3)